runtime/compiler/il/J9DataTypes.cpp

```diff
@@ -127,17 +127,18 @@ const TR_BCDSignCode rawToBCDSignCodeMap[num_raw_bcd_sign_codes] = {
                                                                    bcd_minus,        // raw_bcd_sign_0xd
                                                                    bcd_unsigned      // raw_bcd_sign_0xf
                                                                    };
-
-/// `packedDecimalPrecisionToByteLengthMap` is calculated from the formula:
-///
-///      int32_t n = precision + 1;
-///      if (n is odd)
-///         byteLength = (n/2) + 1;
-///      else
-///         byteLength = (n/2);
-///
-/// EXCEPT for precision=0 that returns a byteLength of zero (a sign code alone
-/// does not a number make)
+/**
+ * `packedDecimalPrecisionToByteLengthMap` is calculated from the formula:
+ *
+ * int32_t n = precision + 1;
+ * if (n is odd)
+ *   byteLength = (n/2) + 1;
+ * else
+ *.  byteLength = (n/2);
```

Comment:
May I ask you to add a few spaces before these lines of example code?  It makes it a little bit easier to read if they are indented.